specification/security/resource-manager/Microsoft.Security/preview/2020-01-01-preview/securityContacts.json

```diff
@@ -0,0 +1,380 @@
+          }
+        },
+        "tags": [
+          "Security Contacts"
+        ],
+        "description": "Get Default Security contact configurations for the subscription",
+        "operationId": "SecurityContacts_Get",
+        "parameters": [
+          {
+            "$ref": "../../../common/v1/types.json#/parameters/ApiVersion"
+          },
+          {
+            "$ref": "../../../common/v1/types.json#/parameters/SubscriptionId"
+          },
+          {
+            "$ref": "#/parameters/SecurityContactName"
+          }
+        ],
+        "responses": {
+          "200": {
+            "description": "OK",
+            "schema": {
+              "$ref": "#/definitions/SecurityContact"
+            }
+          },
+          "default": {
+            "description": "Error response describing why the operation failed.",
+            "schema": {
+              "$ref": "../../../common/v1/types.json#/definitions/CloudError"
+            }
+          }
+        }
+      },
+      "put": {
+        "x-ms-examples": {
+          "Create security contact data": {
+            "$ref": "./examples/SecurityContacts/CreateSecurityContact_example.json"
+          }
+        },
+        "tags": [
+          "Security Contacts"
+        ],
+        "description": "Create security contact configurations for the subscription",
+        "operationId": "SecurityContacts_Create",
+        "parameters": [
+          {
+            "$ref": "../../../common/v1/types.json#/parameters/ApiVersion"
+          },
+          {
+            "$ref": "../../../common/v1/types.json#/parameters/SubscriptionId"
+          },
+          {
+            "$ref": "#/parameters/SecurityContactName"
+          }
+        ],
+        "responses": {
+          "201": {
+            "description": "OK",
+            "schema": {
+              "$ref": "#/definitions/SecurityContact"
+            }
+          },
+          "default": {
+            "description": "Error response describing why the operation failed.",
+            "schema": {
+              "$ref": "../../../common/v1/types.json#/definitions/CloudError"
+            }
+          }
+        }
+      },
+      "delete": {
+        "x-ms-examples": {
+          "Deletes a security contact data": {
+            "$ref": "./examples/SecurityContacts/DeleteSecurityContact_example.json"
+          }
+        },
+        "tags": [
+          "Security Contacts"
+        ],
+        "description": "Security contact configurations for the subscription",
+        "operationId": "SecurityContacts_Delete",
+        "parameters": [
+          {
+            "$ref": "../../../common/v1/types.json#/parameters/ApiVersion"
+          },
+          {
+            "$ref": "../../../common/v1/types.json#/parameters/SubscriptionId"
+          },
+          {
+            "$ref": "#/parameters/SecurityContactName"
+          }
+        ],
+        "responses": {
+          "204": {
```

Comment:
What is your success response code?

https://github.com/Azure/azure-resource-manager-rpc/blob/master/v1.0/resource-api-reference.md#delete-resource

'The resource provider can return 200 (OK) or 204 (NoContent) to indicate that the operation completed successfully. A 200 (OK) should be returned if the object exists and was deleted successfully; and a 204 (NoContent) should be used if the resource does not exist and the request is well formed.'

Reply:
This is the same as our different APIs (as asked from ARM)- 
we are returning 204 with no content for deleting when deletion is successful

Reply:
As per contract (ARM contract) 200 OK must be returned when the resource was successfully deleted and 204 OK in case request is well formed but resource does not exist. can you please fix the behavior in this API and other APIs going forward.

Reply:
thanks for the explanation. Fixed it.